extensions/wikia/Forum/templates/Forum_boardNewThread.php

```diff
@@ -14,7 +14,7 @@
 						<span class="board-list-error"><?= wfMessage( 'forum-no-board-selection-error' )->escaped() ?></span>
 					</div>
 				<? endif; ?>
-				<? if ( $showMiniEditor ): ?>
+				<? if ( $wg->EnableMiniEditorExtForWall ): ?>
 					<?= $app->getView( 'MiniEditorController', 'Header', [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "MiniEditorController" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)